content/en/platform/corda/1.5/cenm/release-notes.md

```diff
@@ -381,7 +381,7 @@ The new Zone Service enables you to store configurations for the Identity Manage
 We have updated the Kubernetes reference deployment to use Helm@3 and to support the new services introduced in CENM 1.3. The updated documentation provides guidance on how to use this deployment with external databases.
 
 #### Configuration obfuscation
-Configuration obfuscation support in CENM 1.3 now involves the use of the [Corda Enterprise Configuration Obfuscator tool](../../../../../en/platform/corda/4.5/enterprise/tools-config-obfuscator.html). Legacy (pre-1.3) obfuscated configurations are still supported, however you should update any such configuration files using the latest version of the Corda Enterprise Configuration Obfuscator tool.
+Configuration obfuscation support in CENM 1.3 now involves the use of the [Corda Enterprise Configuration Obfuscator tool](https://github.com/corda/corda-docs-portal/blob/main/content/en/archived-docs/corda-enterprise/4.5/enterprise/tools-config-obfuscator.md). Legacy (pre-1.3) obfuscated configurations are still supported, however you should update any such configuration files using the latest version of the Corda Enterprise Configuration Obfuscator tool.
```

Comment:
cenm 1.3 followed 4.5 and cenm 1.4 followed 4.6 ... should this (1.5) be following 4.5 as well ?

Reply:
Hi Yariv - that's the 1.5 release notes alright, but the quoted bit is from the 1.3 section, so that link to 4.5 would seem right?